getTotalClaps.js

```diff
@@ -0,0 +1,7 @@
+export const getTotalClaps = story => {
+  let claps = 0;
+  for (const fan of story.fans) {
+    claps += fan.claps.length;
+  }
+  return claps;
+};
```

Comment:
In Javascript, you can achieve the same result by using `.reduce` if you want to accumulate a result from an array. For example:
```
export const getTotalClaps = story => {
  return story.fans.reduce((claps, fan) => claps + fan.claps.length, 0);
};
```